.detekt/detekt.yml

```diff
@@ -79,7 +79,8 @@ complexity:
     threshold: 60
   LongParameterList:
     active: true
-    threshold: 6
```

Comment:
[threshold](https://arturbosch.github.io/detekt/complexity.html#configuration-options-6) was depreciated in a recent update to detekt so I updated it to the `constructorThreshold` and `functionThreshold` with their defaults. Let me know if you'd prefer `constructorThreshold`  to be 6 and I can suppress the warnings in `ObjectPreference` and `EnumPreference`.

Reply:
I think it might be a good time to regenerate a new config file to make sure we're up to date there. Do you mind if I add an extra commit here with this? And I think it's fine to tweak the threshold.

Reply:
Build is passing for me with an untouched brand new detekt config, so it should be straightforward to get that updated.

Reply:
Generated a new default detekt config which passes https://github.com/tfcporciuncula/flow-preferences/pull/5/commits/10ad4e9356a095228d49a18327a2498d6099fb75